plugins/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.tsx

```diff
@@ -88,6 +104,19 @@ const LinearGenomeView = observer(({ model }: { model: LGV }) => {
   const MiniControlsComponent = model.MiniControlsComponent()
   const HeaderComponent = model.HeaderComponent()
 
+  let pinnedTracksTop = VIEW_HEADER_HEIGHT + SCALE_BAR_HEIGHT
+  if (!model.hideHeader) {
+    pinnedTracksTop += HEADER_BAR_HEIGHT
+    if (!model.hideHeaderOverview) {
+      pinnedTracksTop += HEADER_OVERVIEW_HEIGHT
+    }
+  }
+
+  const [pinnedTracks, unpinnedTracks] = partition(
```

Comment:
personal pref/nit: i would use lineargenomeview/model.ts to make a getter for pinned/unpinned tracks with a simple filter like

```
+      /**
+       * #getter
+       */
+      get pinnedTracks() {
+        return self.tracks.filter(t => t.pinned)
+      },
+      /**
+       * #getter
+       */
+      get unpinnedTracks() {
+        return self.tracks.filter(t => !t.pinned)
+      },


```

that a) uses the mobx reactivity and b) might be a bit little simpler than custom partition function